README.md

```diff
@@ -2,197 +2,230 @@
 
-## Login
+      redirects import <csvPath>      Import redirects for the current account and workspace
+      redirects export <csvPath>      Export all redirects in the current account and workspace
+      redirects delete <csvPath>      Delete redirects in the current account and workspace
 
-Start by using `vtex login` to receive your credentials and be able to access other features.
-Simply type this command and follow the instructions.
+      url                               Prints base URL for current account, workspace and account
 
-## Using link
+      workspace                         Alias for vtex workspace info
+      workspace create <name>           Create a new workspace with this name
+      workspace delete <name>           Delete a single or various workspaces
+      workspace info                    Display information about the current workspace
+      workspace list                    List workspaces on this account
+      workspace promote                 Promote this workspace to master
+      workspace reset [name]            Delete and create a workspace
+      workspace use <name>              Use a workspace to perform operations
 
-On an app directory, run `vtex link` and click on or copy the provided URL into your browser.
+    Options:
 
-VTEX Toolbelt will now monitor your files for changes and sync them automatically.
+      -h, --help  show help information
+  ```
+</details>
 
-## Customizing your prompt
+# Second step: login on your VTEX account
 
-You can configure your **terminal prompt** to display relevant information about your current context, like  which **account** you're logged into and which **workspace** you are currently using.
+Start by using `vtex login` to receive your credentials and be able 
+to access other features. Simply type this command and follow the instructions.
 
-Just like knowing which `git` branch you're currently in, having this info in your prompt you help you avoid mistakes and be faster when using VTEX IO.
+<details>
+<summary>Having trouble with login? Follow these steps</summary>
 
-### `bash` users
+You can use `vtex login` to login with your VTEX ID credentials or vtex logout if you're already logged in and want to change credentials.
 
-If you are a _bash_ user you can customize your command prompt by running the following command:
+When logging in, you will be asked for 3 things:
+- The `account` name of the store you wish to work on, that would be your company or clients `account` name;
+- Your VTEX ID email, the one you received access to work with VTEX. Missing the email? Send us a message here.
+- Your VTEX ID password, the one you chose when creating your user. 
 
-```sh
-echo "source $(npm root -g)/vtex/plugins/bash/vtex.bash" >> ~/.bashrc
-echo "source $(npm root -g)/vtex/plugins/bash/prompt.bash" >> ~/.bashrc
-```
+If you wish to work on another `account`, run `vtex logout` and, then `vtex login` again with the one you want to move to. 
```

Comment:
No opinion about suggesting the command `vtex switch`. Would you like to say more about it or suggest how we could do it at the README? That could help me to understand the suggestion

Reply:
`vtex switch <account>` switches the account, so instead of logging out and then logging in you could just switch (and if necessary the toolbelt will redirect you to a login page.
```suggestion
If you wish to work on another `account`, run `vtex switch <account>` specifying the account you want to move to.
```

Reply:
Peeeeerfect! Makes sense